x/distribution/keeper/hooks.go

```diff
@@ -20,11 +20,41 @@ func (h Hooks) AfterValidatorCreated(ctx sdk.Context, valAddr sdk.ValAddress) {
 	h.k.initializeValidator(ctx, val)
 }
 func (h Hooks) BeforeValidatorModified(ctx sdk.Context, valAddr sdk.ValAddress) {
-	val := h.k.stakingKeeper.Validator(ctx, valAddr)
-	// increment period
-	h.k.incrementValidatorPeriod(ctx, val)
 }
 func (h Hooks) AfterValidatorRemoved(ctx sdk.Context, _ sdk.ConsAddress, valAddr sdk.ValAddress) {
+	// force-withdraw commission
+	commission := h.k.GetValidatorAccumulatedCommission(ctx, valAddr)
+	if !commission.IsZero() {
+		coins, remainder := commission.TruncateDecimal()
+
+		// remainder to community pool
+		feePool := h.k.GetFeePool(ctx)
+		feePool.CommunityPool = feePool.CommunityPool.Plus(remainder)
+		h.k.SetFeePool(ctx, feePool)
+
+		// update outstanding
+		outstanding := h.k.GetOutstandingRewards(ctx)
+		h.k.SetOutstandingRewards(ctx, outstanding.Minus(commission))
```

Comment:
side question... outstanding rewards are _where_ the rewards come from when they are withdrawn correct?